numpyro/distributions/continuous.py

```diff
@@ -490,6 +490,51 @@ def icdf(self, q):
         return self.loc - self.scale * jnp.log(-jnp.log(q))
 
 
+class Gompertz(Distribution):
+    arg_constraints = {
+        "loc": constraints.positive,
+        "shape": constraints.positive,
+    }
+    support = constraints.positive
+    reparametrized_params = ["loc", "shape"]
+
+    def __init__(self, loc=1.0, shape=1.0, validate_args=None):
+        self.loc, self.shape = promote_shapes(loc, shape)
+        # Alias parameters as they appear in this article
+        # https://www.tandfonline.com/doi/abs/10.1080/03461238.2012.687697
+        self.__a, self.__b = self.loc, self.shape
```

Comment:
not against but this pattern is not used in the codebase. I guess using the original names in the implementation would be clearer?

Reply:
As a courtesy to users who want to debug their models, we should avoid double underscores and the implied [name mangling](https://stackoverflow.com/questions/1301346).

Reply:
Removed